packers/Dockerfile

```diff
@@ -1,5 +1,9 @@
 FROM debian:buster
 
+# Add bullseye as a secondary APT source
+RUN echo 'deb http://deb.debian.org/debian bullseye main' >> /etc/apt/sources.list.d/bullseye.list
+RUN echo 'APT::Default-Release "buster";' > /etc/apt/apt.conf.d/30default
```

Comment:
Maybe we should add this after the initial install? You know, to limit the blast radius.

Reply:
Actually, even with these lines added, it is rather difficult to pull any stuff from bullseye, hence the version numbers in the RPM install. Setting the default release to 'buster' makes it so that other packages are never candidate unless explicitly requested.